src/parser.rs

```diff
@@ -480,12 +480,20 @@ fn parse_polymorphism_mode(lexer: &mut ParseSession, pou_type: &PouType) -> Opti
 }
 
 fn parse_super_class(lexer: &mut ParseSession) -> Option<String> {
-    if lexer.try_consume(KeywordExtends) {
-        let (name, _) = parse_identifier(lexer)?;
-        Some(name)
-    } else {
-        None
+    let mut extensions = vec![];
+    while lexer.try_consume(KeywordExtends) {
+        let name_and_location = parse_identifier(lexer)?;
+        extensions.push(name_and_location);
     }
+    extensions.iter().skip(1).for_each(|(_, location)| {
+        lexer.accept_diagnostic(
+            Diagnostic::new("Multiple inheritance. POUs can only be extended once.".to_string())
```

Comment:
```suggestion
            Diagnostic::new("Multiple inheritance unsupported, POUs can only be extended once.".to_string())
```
nit: This aligns better with the current error messages we have in place